app/models/concerns/foreman_openscap/hostgroup_extensions.rb

```diff
@@ -16,6 +16,15 @@ def inherited_policies
       end.uniq
     end
 
+    def openscap_proxy
+      return super if ancestry.nil? || self.openscap_proxy_id.present?
+      ::SmartProxy.find_by(:id => inherited_openscap_proxy_id)
+    end
+
+    def inherited_openscap_proxy_id
+      inherited_ancestry_attribute(:openscap_proxy_id)
```

Comment:
Shouldn't this method live in core then? That does not block this PR but such small refactoring would be nice (add it to core and remove from katello and openscap plugins)